### DISCUSSION TITLE
Scroll Restoration

### DISCUSSION THREAD

**Discussion by cloudorbush** [nandorojo/solito#88] [Q&A] [answered]
Hi @nandorojo ,

How would you restore the scroll position of a `ScrollView` or `FlatList` when navigating between pages on web?

I've tried enabling the experimental `scrollRestoration` feature flag in `next.config.js` but that didn't seem to work.

Setting the `scroll` prop to `false` on `<MotiLink />` did not seem to work either.

Is this something I will have to implement manually on every ScrollView/FlatList in my app?

Thanks!

**Comment by nandorojo** [comment, marked as answer]
1. did you read the solito guide on scroll views from the docs? it’s usually best imo to not use a scroll view on the Web side, as explained in that guide.
2. for the times where you do want to use a scroll view, i made this gist: https://gist.github.com/nandorojo/45c00210c8c364017c3ab10a0e94698d
